### PR TITLE
fakemetrics: import 'schemasbackfill' mode

### PR DESCRIPTION
**Pull request by Dieterbe**
from https://github.com/raintank/fakemetrics/pull/20

### CODE REVIEW

File: cmd/mt-fakemetrics/cmd/root.go [hunk 1]
```diff
@@ -86,8 +87,11 @@ var (
 	periodDur  time.Duration
 	flush      int // in ms
 	period     int // in s
+	offset     time.Duration
+	speedup    int
 
 	// global vars
+	outs          []out.Out
```
Comment (Dieterbe) [inline cmd/mt-fakemetrics/cmd/root.go]: @fitzoh this was missing in your import. not sure why?

Comment (Dieterbe) [reply cmd/mt-fakemetrics/cmd/root.go]: aha i see it was unused

File: cmd/mt-fakemetrics/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (fitzoh) [inline cmd/mt-fakemetrics/LICENSE]: My bad on removing the license on the initial import.

Might want to update this portion?

Comment (Dieterbe) [reply cmd/mt-fakemetrics/LICENSE]: this is the template section.

Comment (fitzoh) [reply cmd/mt-fakemetrics/LICENSE]: oops, disregard

**Pull request merged**